test/fixtures/block_zbUPQFA4ybnd8h99KI9Iqh4mogXJibr0syEwuJPrFHhOhld7XBMOUDeXfsIGvYDp.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
Wonder if we need the block in the codebase, or whether we can pick a smaller one?

Reply:
cleared the "poa" field (as it was really huuuuge and not effectively used in test..)